cypress/support/kaoto-ui-commands/editor.js

```diff
@@ -35,6 +38,13 @@ Cypress.Commands.add('editorDeleteLine', (line, repeatCount) => {
         );
 });
 
+Cypress.Commands.add('editorClickUndoXTimes', (times) => {
+    times = times ?? 1;
+    Cypress._.times(times, () => {
```

Comment:
I wonder if we are using lodash anywhere else (we probably are), though it would be good to try to remove it if possible. This would be a good alternative for this method in the future: https://youmightnotneed.com/lodash/#times

Reply:
Thanks, will fix it in the last refactor PR